### PR TITLE
[laa-hmrc-interface-uat] tag resources for AWS access

### PR DESCRIPTION
**Pull request by jsugarman**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/rds.tf [hunk 1]
```diff
@@ -50,6 +50,10 @@ resource "kubernetes_secret" "rds" {
      * url = "postgres://${module.rds.database_username}:${module.rds.database_password}@${module.rds.rds_instance_endpoint}/${module.rds.database_name}"
      *
      */
+
```
Comment (poornima-krishnasamy) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/rds.tf:53]: Please remove the tags as they are supported only for aws resources

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/rds.tf [hunk 2]
```diff
@@ -62,4 +66,8 @@ resource "kubernetes_config_map" "rds" {
     database_name = module.rds.database_name
     db_identifier = module.rds.db_identifier
   }
+
+  tags = {
```
Comment (poornima-krishnasamy) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/rds.tf:70]: Please remove the tags as they are supported only for aws resources

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/s3.tf [hunk 3]
```diff
@@ -45,4 +45,8 @@ resource "kubernetes_secret" "s3-bucket" {
     bucket_arn        = module.s3-bucket.bucket_arn
     secret_access_key = module.s3-bucket.secret_access_key
   }
+
+  tags = {
```
Comment (poornima-krishnasamy) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/s3.tf:49]: Please remove the tags as they are supported only for aws resources

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/elasticache.tf [hunk 4]
```diff
@@ -35,5 +35,9 @@ resource "kubernetes_secret" "elasticache" {
     auth_token               = module.elasticache.auth_token
     redis_url                = "rediss://:${module.elasticache.auth_token}@${module.elasticache.primary_endpoint_address}:6379"
   }
+
```
Comment (poornima-krishnasamy) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/elasticache.tf:38]: Please remove the tags as they are supported only for aws resources

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/ecr.tf [hunk 5]
```diff
@@ -21,4 +21,8 @@ resource "kubernetes_secret" "ecr-repo" {
     access_key_id     = module.ecr-repo.access_key_id
     secret_access_key = module.ecr-repo.secret_access_key
   }
+
```
Comment (poornima-krishnasamy) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/ecr.tf:24]: Please remove the tags as they are supported only for aws resources

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/elasticache.tf (hunk 4 above)
Comment (jsugarman) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-hmrc-interface-uat/resources/elasticache.tf:38]: is this not a resource? I am looking to get read access to elasticache in particular

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**